src/pages/TrueSightV2/components/FeedbackSurvey.tsx

```diff
@@ -64,24 +65,28 @@ const XWrapper = styled.span`
 const LOCALSTORAGE_MODAL_SHOWED = 'showedKyberAIFeedbackSurvey'
 const LOCALSTORAGE_WIDGET_SHOWED = 'showedKyberAIFeedbackSurveyWidget'
 const MOMENT_THIS_SURVEY_RELEASE = 1689768000
+const END_DATE = 1691020800000 // Aug 3
+
 export default function FeedbackSurvey() {
   const [isOpen, setIsOpen] = useState(false)
   const [isOpenWidget, setIsOpenWidget] = useState(false)
   const theme = useTheme()
   const { updatedAt, status } = useGetParticipantKyberAIInfo()
+  const [isShowModalLS, setIsShowModalLS] = useLocalStorage<string | undefined>(LOCALSTORAGE_MODAL_SHOWED, undefined)
+  const [isShowWidgetLS, setIsShowWidgetLS] = useLocalStorage<string | undefined>(LOCALSTORAGE_WIDGET_SHOWED, undefined)
 
   const isValid = useMemo(
-    () => updatedAt < MOMENT_THIS_SURVEY_RELEASE && status === ParticipantStatus.WHITELISTED,
+    () => updatedAt < MOMENT_THIS_SURVEY_RELEASE && status === ParticipantStatus.WHITELISTED && Date.now() < END_DATE,
     [updatedAt, status],
   )
 
   useEffect(() => {
     if (!isValid) return
-    if (!localStorage.getItem(LOCALSTORAGE_MODAL_SHOWED)) {
+    if (!isShowModalLS) {
       setIsOpen(true)
-      localStorage.setItem(LOCALSTORAGE_MODAL_SHOWED, '1')
+      setIsShowModalLS('1')
     }
-    if (!localStorage.getItem(LOCALSTORAGE_WIDGET_SHOWED)) {
+    if (!isShowWidgetLS) {
       setIsOpenWidget(true)
     }
   }, [isValid])
```

Comment:
React Hook useEffect has missing dependencies: 'isShowModalLS', 'isShowWidgetLS', and 'setIsShowModalLS'. Either include them or remove the dependency array